tests/ignite/metrics/nlp/test_bleu.py

```diff
@@ -131,14 +230,14 @@ def test_bleu():
     bleu.update(([corpus.cand_2a], [corpus.references_2]))
     bleu.update(([corpus.cand_2b], [corpus.references_2]))
     bleu.update(([corpus.cand_3], [corpus.references_2]))
-    value = bleu._corpus_bleu([corpus.references_1], [corpus.cand_1])
-    value += bleu._corpus_bleu([corpus.references_2], [corpus.cand_2a])
-    value += bleu._corpus_bleu([corpus.references_2], [corpus.cand_2b])
-    value += bleu._corpus_bleu([corpus.references_2], [corpus.cand_3])
+    value = bleu._sentence_bleu(corpus.references_1, corpus.cand_1)
```

Comment:
Could we rename `test_bleu` in `test_accumulation_macro_bleu` and add `test_accumulation_micro_bleu` ?